libs/shared-atomic-design-components/src/lib/atoms/Checkbox.stories.tsx

```diff
@@ -1,18 +1,24 @@
-import { Meta } from '@storybook/react/types-6-0'
+import { ComponentMeta } from '@storybook/react';
 import { storybookTemplate } from '../helpers/StorybookTemplate'
 
 import CheckboxComponent from './Checkbox'
-import 'bulma/css/bulma.min.css'
 
 export default {
   title: 'atoms/Checkbox',
   component: CheckboxComponent,
-} as Meta
+  argTypes: {
+    handleChange: {
+      control: { type: 'object' }
+    },
+    handleBlur: {
+      control: { type: 'object' }
+    }
+  }
```

Comment:
What are these for? Is there any documentation I can check to learn this approach?